### PR TITLE
Add support for passing clippy arguments in comment-able file

### PR DESCRIPTION
**Pull request by gbhand**
# Background

As described in [monorail#1298](https://github.com/parallelsystems/monorail/issues/1298), larger-scale CI workflows (often using >100 clippy arguments) are difficult to maintain and understand without the ability to add comments.

# Description

Added support for reading clippy args from an `sh`-style file specified by the `args-file` parameter. Each individual argument may be on same or different lines. Full-line comments are supported for better organization and clarification. Inline comments are not supported.

# Verification

* Success in test run https://github.com/parallelsystems/monorail/runs/7307071120?check_suite_focus=true#step:6:3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,35 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          components: clippy
+          override: true
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args-file: "./clippy_args.sh"
```
Comment (joshparallel) [inline README.md]: Maybe the extension should be e.g. `.txt` since it's not an executable shell script?

File: README.md [hunk 2]
```diff
@@ -60,10 +89,13 @@ jobs:
 | `args`      |          | Arguments for the `cargo clippy` command                                                                                               | string |         |
 | `use-cross` |          | Use [`cross`](https://github.com/rust-embedded/cross) instead of `cargo`                                                               | bool   | false   |
 | `name`      |          | Name of the created GitHub check. If running this action multiple times, each run must have a unique name.                             | string | clippy  |
+| `args-file` |          | Path to file containing line-delimited arguments for `cargo clippy` (`sh`-style comments supported) | string | |
 
 For extra details about the `toolchain`, `args` and `use-cross` inputs,
 see [`cargo` Action](https://github.com/actions-rs/cargo#inputs) documentation.
 
+**NOTE**: only specify one argument souce, either `args` or `args-file`.
```
Comment (joshparallel) [inline README.md]:
```suggestion
**NOTE**: only specify one argument source, either `args` or `args-file`.
```

File: src/input.ts [hunk 3]
```diff
@@ -23,12 +24,14 @@ export function get(): Input {
     }
     const useCross = input.getInputBool('use-cross');
     const name = input.getInput('name');
+    const argsFilePath = input.getInput('args-file')
 
     return {
-        token: input.getInput('token', {required: true}),
+        token: input.getInput('token', { required: true }),
```
Comment (joshparallel) [inline src/input.ts]: Might be good to avoid changing formatting to avoid pain in merges/rebases if we ever want to get upstream changes?

**Review by joshparallel — COMMENTED** [review]
Looks great! Couple tiny comments.

File: src/input.ts (hunk 3 above)
Comment (gbhand) [reply src/input.ts]: Good idea. Changed.

**Review by joshparallel — APPROVED** [review]
LGTM!

**Pull request merged**